src/PHPCR/Util/Console/Command/WorkspaceImportCommand.php

```diff
@@ -58,11 +66,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $session->importXML(
-            $parentPath,
-            $filename,
-            ImportUUIDBehaviorInterface::IMPORT_UUID_CREATE_NEW
-        );
+        $uuidBehavior = $input->getOption('uuid-behavior');
+        if (!array_key_exists($uuidBehavior, self::UUID_BEHAVIOR)) {
+            $output->writeln(sprintf('<error>UUID-Behavior "%s" is not supported</error>', $uuidBehavior));
```

Comment:
can we please add another line saying `'Supported behaviors are %s', implode(', ', self::UUID_BEHAVIOR)`